### PR TITLE
LMN-1159 Add price row variant

### PR DESCRIPTION
**Pull request by marianeum**
Added a new row variant for the price component & update demo app to match Figma

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: app/src/main/java/net/skyscanner/backpack/demo/compose/PriceStory.kt [hunk 1]
```diff
@@ -38,98 +37,63 @@ import net.skyscanner.backpack.demo.meta.ComposeStory
...
-@ComposeStory("Extra Small")
-fun PriceXSStory(modifier: Modifier = Modifier) =
-    PriceDemo(BpkPriceSize.ExtraSmall, modifier)
-
-@Composable
-private fun PriceDemo(
-    size: BpkPriceSize,
-    modifier: Modifier = Modifier,
-) {
+@ComposeStory
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/compose/PriceStory.kt:40]: just simplified this a lot to match what figma shows

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/BpkPrice.kt [hunk 2]
```diff
@@ -22,10 +22,12 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import net.skyscanner.backpack.compose.price.internal.BpkPriceAlignEnd
 import net.skyscanner.backpack.compose.price.internal.BpkPriceAlignStart
+import net.skyscanner.backpack.compose.price.internal.BpkPriceRow
 
 enum class BpkPriceAlign {
     Start,
     End,
+    Row,
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/BpkPrice.kt:30]: there's now a new row variant

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceAlignEnd.kt [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import net.skyscanner.backpack.compose.price.BpkPriceSize
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.utils.applyIf
+
+@Composable
+internal fun BpkPriceAlignEnd(
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceAlignEnd.kt:35]: just extracted to a separate file - no changes made here

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceRow.kt [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextDecoration
+import net.skyscanner.backpack.compose.price.BpkPriceSize
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+
+@Composable
+internal fun BpkPriceRow(
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceRow.kt:32]: that's the new variant- mainly just rearranging the existing components/styles

File: backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceStyles.kt [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+package net.skyscanner.backpack.compose.price.internal
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.TextStyle
+import net.skyscanner.backpack.compose.price.BpkPriceSize
+import net.skyscanner.backpack.compose.theme.BpkTheme
+
+@Composable
+internal fun BpkPriceSize.mainTextStyle(): TextStyle =
```
Comment (marianeum) [inline backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/internal/BpkPriceStyles.kt:27]: again just moved to a separate file

File: docs/compose/Price/README.md [hunk 6]
```diff
@@ -4,24 +4,11 @@
...
-
-## Large
-
-| Day                                                                                                                                                         | Night                                                                                                                                                                      |
-|-------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Price/screenshots/large.png" alt="Price component" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/Price/screenshots/large_dm.png" alt="Price component - dark mode" width="375" /> |
+## Default
```
Comment (marianeum) [inline docs/compose/Price/README.md:7]: there's just one screenshot now (same as figma)

**Review by xo0ps — APPROVED** [review]

**Pull request merged**